x-pack/solutions/search/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/pipelines.tsx

```diff
@@ -65,6 +65,16 @@ export const SearchIndexPipelines: React.FC = () => {
   const { makeRequest: revertPipeline } = useActions(RevertConnectorPipelineApilogic);
   const apiIndex = isApiIndex(index);
   const extractionDisabled = getContentExtractionDisabled(index);
+  const buttonRef = useRef<HTMLButtonElement | null>(null);
+
+  const onCloseHandler = () => {
```

Comment:
nit: This name could be more descriptive for what it's used for `onCloseDeleteModal`